e2e/test/scenarios/collections/personal-collections.cy.spec.js

```diff
@@ -96,7 +96,7 @@ describe("personal collections", () => {
       // Check that it's not possible to open permissions modal via URL for personal collection
       // cy.location().then(location => {
       //   cy.visit(`${location}/permissions`);
-      //   cy.get(".Modal").should("not.exist");
+      //   modal().should("not.exist");
```

Comment:
even though this is just a comment, it's worth changing it in case we want to uncomment this at some point in the future 🤷